### DISCUSSION TITLE
Installing on Ubuntu server

### DISCUSSION THREAD

**Discussion by thecivvie** [filipsPL/autowx2#107] [Q&A] [answered]
I am moving mine from a Pi 3B to an atom based Linux box. Any suggestions or tips on setup or should I use Debian so close to the Pi os

**Comment by SA7BNT** [comment, marked as answer]
> I am moving mine from a Pi 3B to an atom based Linux box. Any suggestions or tips on setup or should I use Debian so close to the Pi os

Running Debian 10 on an x86. No problem with that.

**Reply by thecivvie** [reply]
OK, that sounds the best idea. It is new to me, only used SuSe and Ubuntu before.

**Comment by thecivvie** [comment]
I am getting an error installing due to bidict only being 0.18.4. I installed 0.21.2 but it is Python 3. back to the drawing board
